frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/viewProperty.component.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Controller } from 'react-hook-form';
+import { PropertyProps } from './properties.types';
+import { TicketView } from './ticketView/ticketView.component';
+
+export const ViewProperty = ({
+	property: { name, required },
+	formError,
+	defaultValue,
+	...props
```

Comment:
You could do the same as you did for `ImageProperty`, i.e. to convert `property.name` to `label` and then pass the `name` directly without calling `name={props.name}`
```ts 
export const ViewProperty = ({
        property: { name: label, readOnly, required }, // <- name: label
	defaultValue,
	formError,
	name, // <- also this one
```